hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -58,11 +58,15 @@
         },
         {
           "name": "CryptoTransfer",
-          "baseFee": 18,
-          "extras": [
-            {  "name": "KEYS",       "includedCount": 1 },
-            {  "name": "ACCOUNTS",       "includedCount": 1 },
-            {  "name": "STANDARD_FUNGIBLE_TOKENS", "includedCount": 1}
+          "baseFee": 0,
+          "extras": [
+            {  "name": "ACCOUNTS",                       "includedCount": 2 },
+            {  "name": "STANDARD_FUNGIBLE_TOKENS",       "includedCount": 1 },
+            {  "name": "STANDARD_NON_FUNGIBLE_TOKENS",   "includedCount": 1 },
```

Comment:
Shouldn't these be defaults based on the `CryptoTransfer + subtype` transaction?